wordlist.py

```diff
@@ -49,6 +49,51 @@ class MissingTranslationException(Exception):
             'mean',
             'poor',
         ]
+    ],
+	"de_DE": [
+        [
+            'Der Raum',
+            'Bytespeicher'
+        ],
+        [
+            'ist',
+        ],
+        [
+            'offen',
+            'Zutritt möglich',
```

Comment:
This would be "Der Raum ist Zutritt moeglich"... The final message is one choice of each list, meaning it would be okay to leave out choices or change them dramatically rather than just translating them word-by-word.